sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/ui/HiveThriftServer2Listener.scala

```diff
@@ -218,7 +232,9 @@ private[thriftserver] class HiveThriftServer2Listener(
         executionData.state = ExecutionState.CLOSED
         updateStoreWithTriggerEnabled(executionData)
         executionList.remove(e.id)
-      case None => logWarning(s"onOperationClosed called with unknown operation id: ${e.id}")
+      case None => logWarning(
+        log"onOperationClosed called with unknown operation id: ${MDC(OP_ID, e.id)}"
```

Comment:
From the call side, the id is statement_id

Reply:
That means that we should change all `OP_ID` into `STATEMENT_ID` from this file, right?

Reply:
Probably, please double check the callsites to make sure we are using the right variable name.